versionhistory.md

```diff
@@ -0,0 +1,78 @@
+# Version history endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint represents all related versions.
+
+Whether version history information is accessible depends on versioning.item.history.view.admin configuration
+If this is set to true, the version history can only be retrieved if the user is an admin of the last version's item
+
+## Get version history
+
+**GET /api/versioning/versionhistories/<:versionHistoryId>**
+
+Provide version information for the version history id.
+
+```json
+{
+  "id": "1",
+  "type": "versionhistory",
+  "_links": {
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistory/1"
```

Comment:
please fix the plural also in the examples

Reply:
This has been updated